### PR TITLE
In processing refactoring.

### PR DESCRIPTION
**Pull request by MiqayelChaloyan**

### CODE REVIEW

File: screens/FeedScreen/index.js [hunk 1]
```diff
@@ -84,3 +90,14 @@ FeedScreen.propTypes = {
 };
 
 export default FeedScreen;
+
+{
+  /* TODO: This part is for a test and will be changed lately.*/
+}
+{
```
Comment (Varuj2002) [inline screens/FeedScreen/index.js:97]: ?

File: screens/FeedScreen/FeedItem.jsx [hunk 2]
```diff
@@ -3,43 +3,52 @@ import {Text, View, Image, TouchableOpacity} from 'react-native';
...
-        }}>
-        <View style={styles.contentContainer}>
-          <View style={styles.imageContainer}>
-            <Image source={{uri: item.imageUri}} style={styles.feedItemImage} />
-          </View>
-          <View style={styles.contentInfo}>
-            <View style={styles.headerContent}>
-              <Text style={styles.headerFeedText}>{item.title}</Text>
-              <Text style={styles.aboutContentDate}>{diffMonths}m ago</Text>
+    item && (
```
Comment (Varuj2002) [inline screens/FeedScreen/FeedItem.jsx:18]: if(!item){
return;
}

Comment (Varuj2002) [inline screens/FeedScreen/FeedItem.jsx:18]: an empty object is always true, move to item.title (for example)

File: screens/ProfileScreen/profile/index.js [hunk 3]
```diff
@@ -61,9 +75,27 @@ const Profile = ({navigation}) => {
       navigation.navigate('Images');
     }
     return () => setAnswer('');
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (Varuj2002) [inline screens/ProfileScreen/profile/index.js:78]: ?

File: screens/UsersScreen/index.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            }
+          />
+        </View>
+        <TextInput
+          name="search"
+          placeholder="Search"
+          placeholderTextColor={theme.colors.dark_gray}
+          style={styles.input}
+          variant="outlined"
+          // onChangeText={value => setSearchItemValue(value)}
```
Comment (Varuj2002) [inline screens/UsersScreen/index.js:47]: ??

File: screens/FeedScreen/index.js (hunk 1 above)
Comment (MiqayelChaloyan) [reply screens/FeedScreen/index.js:97]: khanem Varuj jan

File: screens/UsersScreen/style.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
+import {StyleSheet} from 'react-native';
+import {
+  horizontalScale,
+  moderateScale,
+  verticalScale,
+} from '../../assets/metrics/Metrics';
+import {theme} from '../../assets/theme/theme';
+
+const styles = StyleSheet.create({
+  //index.js
```
Comment (Varuj2002) [inline screens/UsersScreen/style.js:10]: ?

File: screens/authentication/login/style.js [hunk 6]
```diff
@@ -36,7 +36,7 @@ const styles = StyleSheet.create({
     color: theme.colors.black,
     textAlign: 'center',
     lineHeight: verticalScale(36.31),
-    left: verticalScale(10),
+    // left: verticalScale(10),
```
Comment (Varuj2002) [inline screens/authentication/login/style.js:39]: ??????

File: screens/ProfileScreen/profile/index.js (hunk 3 above)
Comment (MiqayelChaloyan) [reply screens/ProfileScreen/profile/index.js:78]: I found a temporary solution.

File: screens/UsersScreen/index.js (hunk 4 above)
Comment (MiqayelChaloyan) [reply screens/UsersScreen/index.js:47]: I corrected.

File: screens/ContentScreen/index.js [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+    });
+
+    return () => {
+      showSubscription.remove();
+      hideSubscription.remove();
+    };
+  }, []);
+
+  return (
+    feeds && (
```
[the remark this answers is not in the document]
Comment (MiqayelChaloyan) [reply screens/ContentScreen/index.js:38]: I corrected.

File: screens/UsersScreen/style.js (hunk 5 above)
Comment (MiqayelChaloyan) [reply screens/UsersScreen/style.js:10]: I will fix it immediately after connecting to the main.

File: screens/authentication/login/style.js (hunk 6 above)
Comment (MiqayelChaloyan) [reply screens/authentication/login/style.js:39]: I corrected.

**Pull request merged**